core/aws-core/src/main/java/software/amazon/awssdk/awscore/interceptor/TracingSystemSetting.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package software.amazon.awssdk.awscore.interceptor;
```

Comment:
Can we move to `software.amazon.awssdk.awscore.internal.interceptor`?

Reply:
Just to make sure i understand correctly - you want me to move only TracingSystemSetting to software.amazon.awssdk.awscore.internal.interceptor correct?